pkg/packagekit/updates.jsx

```diff
@@ -245,6 +266,39 @@ function UpdatesList(props) {
     );
 }
 
+function UpdateHistory(props) {
+    if (!props.history)
+        return null;
+
+    var now = Date.now();
+
+    function formatHeading(time) {
+        if (time)
+            return cockpit.format(_("The following packages were updated $0 ago:"), moment.duration(now - time).humanize());
```

Comment:
Is our moment js locale aware? The library does have `fromNow()` that would eliminate the need to use `now`